modules/navier_stokes/test/tests/finite_volume/ins/natural_convection/diff_heated_cavity_linear_segregated.i

```diff
@@ -0,0 +1,250 @@
+################################################################################
+# MATERIAL PROPERTIES
+################################################################################
+rho = 3279.
+T_0 = 875.0
+mu = 1.
+k_cond = 38.0
+cp = 640.
+alpha = 3.26e-4
+
+walls = 'right left top bottom'
+
+[GlobalParams]
+  rhie_chow_user_object = 'ins_rhie_chow_interpolator'
+  advected_interp_method = 'upwind'
+  u = superficial_vel_x
+  v = superficial_vel_y
+[]
+
+[Problem]
+  linear_sys_names = 'u_system v_system pressure_system energy_system'
+  previous_nl_solution_required = true
+[]
+
+################################################################################
+# GEOMETRY
+################################################################################
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0
+    xmax = 1
+    ymin = 0
+    ymax = 1
+    nx = 40
+    ny = 40
```

Comment:
Let's make this a little smaller for the same reasons I mentioned above. We can add bigger tests with measurements in the flow test bed very soon.

Reply:
done